app/helpers/currency-symbol.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+import { paymentCurrencies } from 'open-event-frontend/utils/dictionary/payment';
+
+const { Helper } = Ember;
+
+function getObject(element) {
+  return element.code === String(this);
+}
+export function currencySymbol(params) {
+  return paymentCurrencies.find(getObject, params).symbol;
```

Comment:
Can you use lodash's find instead ? (https://lodash.com/docs/4.17.4#find) ... It's similar as far as perf is considered, but easier to read.